docs/articles/new-expensify/connections/netsuite/Connect-To-NetSuite.md

```diff
@@ -1,162 +1,183 @@
-2. Click **Customize** or **Edit** next to the Standard Expense Report form, then click _Screen Fields > Main_.
-3. Verify the “Created From” label has “Show” checked and the "Display Type" is set to "Normal."
-4. Click the second sub-header, **Expenses**, and verify that the "Show" column for "Receipt URL" is checked.
-5. Go to _Customization > Forms > Transaction Forms_ and ensure that all other transaction forms with the expense report type have this same configuration.
+1. Go to **Customization > Forms > Transaction Forms**
+2. Edit the **Standard Expense Report** form:
+   - Under **Main** tab: 
+     - **Created From** should be shown
+   - Under **Expenses** subtab:
+     - **Receipt URL** should be shown
+3. Repeat for all expense report-type forms
 
+---
 
-# Step 9: Confirm Vendor Bill Transactions Forms are Configured Properly
+# Step 9: Check Vendor Bill transaction forms
 
-1. In NetSuite, go to _Customization > Forms > Transaction Forms_.
-2. Click **Customize** or **Edit** next to your preferred Vendor Bill form.
-3. Click _Screen Fields > Main_ and verify that the “Created From” label has “Show” checked and that Departments, Classes, and Locations have the “Show” label unchecked.
-4. Under the **Expenses** sub-header (make sure to click the “Expenses” sub-header at the very bottom and not “Expenses & Items”), ensure “Show” is checked for Receipt URL, Department, Location, and Class.
-5. Go to _Customization > Forms > Transaction Forms_ and ensure that all other transaction forms with the vendor bill type have this same configuration.
+1. Go to **Customization > Forms > Transaction Forms**
+2. Edit your Vendor Bill form:
+   - Under **Main** tab:
+     - Show “Created From”
+     - Hide **Department**, **Class**, and **Location**
+   - Under **Expenses** subtab:
+     - Show **Receipt URL**, **Department**, **Location**, **Class**
+3. Repeat for all vendor bill-type forms
 
+---
 
-# Step 10: Confirm Vendor Credit Transactions Forms are Configured Properly
+# Step 10: Check Vendor Credit transaction forms
 
-1. In NetSuite, go to _Customization > Forms > Transaction Forms_.
-2. Click **Customize** or **Edit** next to your preferred Vendor Credit form, then click _Screen Fields > Main_ and verify that the “Created From” label has “Show” checked and that Departments, Classes, and Locations have the “Show” label unchecked.
-3. Under the Expenses sub-header (make sure to click the “Expenses” sub-header at the very bottom and not “Expenses & Items”), ensure “Show” is checked for Receipt URL, Department, Location, and Class.
-4. Go to _Customization > Forms > Transaction Forms_ and ensure that all other transaction forms with the vendor credit type have this same configuration.
+1. Go to **Customization > Forms > Transaction Forms**
+2. Edit your Vendor Credit form:
+   - Under **Main** tab:
+     - Show “Created From”
+     - Hide **Department**, **Class**, and **Location**
+   - Under **Expenses** subtab:
```

Comment:
I think "subtab" is a valid word: https://en.wiktionary.org/wiki/subtab

Investigating what to do with this spellcheck error.

Reply:
Asking here: https://expensify.slack.com/archives/C01GTK53T8Q/p1749578435882889